### ISSUE TITLE
Bug: CSS descriptor "unicode-range" causes runtime error with rule "css/no-invalid-at-rules"

### ISSUE THREAD

**Issue by Otlikov** [eslint/css#201]
### Environment

ESLint version: 9.31.0
@eslint/css version: 0.10.0
Node version: 24.4.0
npm version: 11.4.2
Operating System: macOS 15.5 (24F74)

### Which language are you using?

stylesheet

### What did you do?

<details>
<summary>Configuration</summary>

```
css/css/no-invalid-at-rules: ["error"]
```
</details>

```css
/* eslint css/css/no-invalid-at-rules: ["error"] */

@font-face {
    font-family: 'myFont';
    src: url(./myFont.woff2) format('woff2');
    unicode-range: U+00, U+0D;
}

```

### What did you expect to happen?

CSS descriptor "unicode-range" should be handled properly.

### What actually happened?

Runtime error occured with:

```
Oops! Something went wrong! :(

ESLint: 9.31.0

Error: Bad syntax reference: <unicode-range-token>
Occurred while linting...
```

### Link to Minimal Reproducible Example

none

### Participation

- [ ] I am willing to submit a pull request for this issue.
